### DISCUSSION TITLE
TuringCourses/major_module/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#56] [General]
# TuringCourses/major_module/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_module/

**Comment by HobbitQia** [comment]
注意，CS 培养方案里的系统模块的课程中，并行计算与多核编程、计算机系统综合实现、计算机系统综合实现从没开过课，因此想选择系统模块的同学只能汇编与接口+嵌入式系统了[ac01]
